src/test/kotlin/no/nav/familie/ef/personhendelse/inntekt/InntektClientTest.kt

```diff
@@ -116,4 +140,326 @@ class InntektClientTest {
+                                        "leveringstidspunkt": "2022-01",
+                                        "opptjeningsland": "NO",
+                                        "utbetaltIMaaned": "2020-04",
+                                        "opplysningspliktig": {
+                                            "identifikator": "928497704",
+                                            "aktoerType": "ORGANISASJON"
+                                        },
+                                        "virksomhet": {
+                                            "identifikator": "947064649",
+                                            "aktoerType": "ORGANISASJON"
+                                        },
+                                        "tilleggsinformasjon": {
+                                            "kategori": "Nettoloennsordning"
+                                        },
+                                        "inntektsmottaker": {
+                                            "identifikator": "07097614819",
+                                            "aktoerType": "NATURLIG_IDENT"
+                                        },
+                                        "inngaarIGrunnlagForTrekk": true,
+                                        "utloeserArbeidsgiveravgift": true,
+                                        "informasjonsstatus": "InngaarAlltid",
+                                        "beskrivelse": "fastloenn",
+                                        "skatteOgAvgiftsregel": "nettoloenn"
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                },
+                "2020-05": {
+                    "928497704": [
+                        {
+                            "versjon": 2,
+                            "opplysningspliktig": "928497704",
+                            "virksomhet": "947064649",
+                            "innleveringstidspunkt": "2021-12-23T11:40:27.177",
+                            "arbeidsInntektInformasjon": {
+                                "inntektListe": [
+                                    {
+                                        "inntektType": "LOENNSINNTEKT",
+                                        "beloep": 35000,
+                                        "fordel": "kontantytelse",
+                                        "inntektskilde": "A-ordningen",
+                                        "inntektsperiodetype": "Maaned",
+                                        "inntektsstatus": "LoependeInnrapportert",
+                                        "leveringstidspunkt": "2022-01",
+                                        "opptjeningsland": "NO",
+                                        "utbetaltIMaaned": "2020-05",
+                                        "opplysningspliktig": {
+                                            "identifikator": "928497704",
```

Comment:
Burde vi bruke dummy-orgnr? :P Sånn att ikke folk finner denne koden når de søker etter 928497704? 😁

Reply:
Orgnumrene var faktisk generert fra Dolly, så det viser seg at de kan bruke reelle orgnumre 😅 Det er sikkert bevisst valg, fordi det er ikke data som er sensitive, men jeg endret om til dummy orgnumre for sikkerhetsskyld.